types/src/v0/v0_3/stake_table.rs

```diff
@@ -10,3 +10,17 @@ pub struct PermissionedStakeTableEntry(NodeInfoJf);
 /// Stake table holding all staking information (DA and non-DA stakers)
 #[derive(Debug, Clone, Serialize, Deserialize, From)]
 pub struct CombinedStakeTable(Vec<PeerConfigKeys<PubKey>>);
+
+#[derive(Clone, Debug, From, Into)]
+/// NewType to disambiguate DA Membership
+pub struct DAMembers(pub Vec<StakeTableEntry<PubKey>>);
+
+#[derive(Clone, Debug, From, Into)]
+/// NewType to disambiguate StakeTable
+pub struct StakeTable(pub Vec<StakeTableEntry<PubKey>>);
+
+#[derive(Clone, Debug)]
+pub struct StakeTables {
```

Comment:
I'm thinking we still have some unneeded complexity here to to maintaining to separate vectors..

Reply:
I think its fine to have one vec. The only downside would be to filter the entries for da_stake_table() which is fine IMO